emission/core/wrapper/suggestion_sys.py

```diff
@@ -129,7 +129,7 @@ def business_details(api_key, business_id):
 """
 def return_address_from_location_nominatim(lat, lon):
     geocode_obj = geo.Geocoder()
-    return geocode_obj.reverse_geocode(lat, lon)
+    return geocode_obj.reverse_geocode(float(lat), float(lon))
```

Comment:
what did you need this for? Was there a change to the `reverse_geocode` API?

Reply:
It's needed because it throws a syntax error saying that the lat and lon passed is a string.

Reply:
ok so I don't know how that ever worked. but I don't have time to dig deeper right now.